app/avo/actions/upload_versions_file.rb

```diff
@@ -0,0 +1,18 @@
+class UploadVersionsFile < BaseAction
+  self.name = "Upload Versions File"
+  self.visible = lambda {
+    current_user.team_member?("rubygems-org") && view == :index
+  }
+  self.standalone = true
+  self.confirm_button_label = "Upload"
+
+  class ActionHandler < ActionHandler
+    def handle_standalone
+      UploadVersionsFileJob.perform_later
+
+      succeed("Upload job scheduled")
+
+      Version.last
```

Comment:
Any reason to return this?

Reply:
need to return an Auditable for creating the Audit